meta/runtime.yml

```
@@ -59,3 +59,72 @@ plugin_routing:
       deprecation:
         removal_date: '2023-01-01'
         warning_text: Use 'ansible.utils.cli_parse' module instead.
+  filter:
+    cidr_merge:
+      deprecation:
+        removal_date: '2024-01-01'
+        warning_text: Use 'ansible.utils.cidr_merge' module instead.
```

Shouldn't these have redirects?
```suggestion
        warning_text: Use 'ansible.utils.cidr_merge' module instead.
      redirect: ansible.utils.cidr_merge
```